src/modules/sync/lib/tree-utils.ts

```diff
@@ -27,6 +27,10 @@ export const shouldLimitDepth = ( relativePath: string ): boolean => {
 		return true;
 	}
 
```

Comment:
Maybe we could add a short comment above to be consistent with other similar checks above.